### PR TITLE
Move Confluence pre-release highlights into permanent home

### PR DESCRIPTION
**Pull request by cwickham**
Will close quarto-dev/quarto-cli#4892

### CODE REVIEW

File: docs/publishing/index.qmd [hunk 1]
```diff
@@ -10,13 +10,14 @@ There are a wide variety of ways to publish documents, presentations, and websit
...
 | [Posit Connect](rstudio-connect.qmd) | Publishing platform for secure sharing of data products within an organization. Use Posit Connect when you want to publish content within an organization rather than on the public internet. |
-| [Netlify](netlify.qmd)                 | Professional web publishing platform. Use Netlify when you want support for custom domains, authentication, previewing branches, and other more advanced capabilities.                          |
-| [Other Services](other.qmd)            | Content rendered with Quarto uses standard formats (HTML, PDFs, MS Word, etc.) that can be published anywhere. Use this if one of the methods above don't meet your requirements.               |
+| [Netlify](netlify.qmd)               | Professional web publishing platform. Use Netlify when you want support for custom domains, authentication, previewing branches, and other more advanced capabilities.                        |
+| [Confluence](confluence.qmd)         | Publishing platform for supporting team collaboration. Use Confluence when you want to control access through Confluence's spaces.                                                            |
```
Comment (cwickham) [inline docs/publishing/index.qmd:19]: @allenmanning Can you take a look at the wording here? Every other line in this table has a "Use this when..." sentence. What should we say for Confluence?

Comment (allenmanning) [reply docs/publishing/index.qmd:19]: I think this works well.  I would also like to suggest "when you want to use Confluence to share documents in team Spaces".

File: docs/output-formats/all-formats.qmd [hunk 2]
```diff
@@ -68,26 +68,27 @@ See below for a list of all output formats by type along with links to their ref
...
+| [InDesign](../reference/formats/icml.qmd)               | ICML is an XML representation of an Adobe InDesign document.                                                                                                                                   |
+| [TEI Simple](../reference/formats/tei.qmd)              | TEI Simple aims to define a new *highly-constrained* and *prescriptive* subset of the Text Encoding Initiative (TEI) Guidelines suited to the representation of early modern and modern books. |
+| [FictionBook](../reference/formats/fb2.qmd)             | FictionBook is an open XML-based e-book format.                                                                                                                                                |
+| [Confluence](../reference/formats/confluence-html.qmd)  | The Confluence format is a variation of XHTML used to store the content of pages on the Confluence platform.                                                                                   |
```
[the remark this answers is not in the document]
Comment (allenmanning) [reply docs/output-formats/all-formats.qmd:91]: Looks great.

**Comment by jjallaire** [comment]
I just created a PR for the v1.3 RC changes to the website (so once this is merged it will also show up in that preview)

**Comment by cwickham** [comment]
> Is this ready to be moved out of Draft? 

Not quite...need to sort out the reference section for `confluence-html`. @dragonstyle is helping.

File: docs/output-formats/all-formats.qmd (hunk 2 above)
[the remark this answers is not in the document]
Comment (cwickham) [reply docs/output-formats/all-formats.qmd]: Removed for now until `confluence-html` is in schema

**Pull request merged**